_posts/2016-08-25-hybox-names.markdown

```diff
@@ -0,0 +1,38 @@
+
+    * Hydra-in-a-Box and HyBox both seem poor choices here: the former is clunky and overly long. The latter is perhaps too insider-y and lacks the natural power and mass-market appeal that a name like "Avalon" brings.
+
+    * Also, Hydra-in-a-Box implies the full functionality and flexibility of the Hydra technical framework is present in the product, but all nicely wrapped up in one package. This is not true, as we’ll be making opinionated selections, based on the community’s requirements, about functionality to include. As such, we’re looking for a name that connotes one is getting a particular flavor or profile of Hydra.
+
+    * We may want to pick up on the useful trend of picking a name related to or derived from "Hydra" in some way, but distinct. Hydrangea, Hypatia, Hydraulics, Hyperspace all fall into this pattern. 
+
+    * We need to be sure to clear the obvious hurdles--no similar products with similar names, no unintentionally offensive meanings in other languages, available domain names, etc.
+
+In short, we’re looking for a product name **that has strong branding potential, will have mass market appeal, and is distinct from but is easily related to Hydra. **
+
+For now, we have started by creating a new component that augments the Hydra stack with turnkey features. The application is, fittingly (we think), named *Lerna* after the lake where the mythological Hydra lived (special thanks to Rick Johnson at Notre Dame, the first person to suggest this name).  
+
+4. **hosted service. **One of the goals of the project is to produce a "cloud-ready" version of the repository product, so that sites can make use of the platform without having to install it locally. The project partners intend to pilot a hosted service next year to kickstart this process. (The cloud-ready components and configurations will also be open source, and a good number of other sites have been in touch about potentially running this for local consortia.) We have time and space to think about this name, and will get there next year.
+
+5. **DPLA ****metadata aggregator product**. While a better and easy-to-install repository is a key component of the grant, it’s not the only one. The other major technical focus is to enhance DPLA’s metadata tooling to get more, better and richer metadata flowing among systems. This will elevate DPLA, its hubs, contributing member institutions, and users by raising the bar on discovery; it will also help any environment where metadata syndication, aggregation, and indexing is an essential component for improving discovery. This product is intended to build on [Krikri](https://dp.la/info/2015/02/11/dpla-releases-krikri-0-1-3/), a software component used to build DPLA’s metadata ingestion system. Because this will be a general-purpose tool, we expect this product to have its own name, branding, and identity (likely very strongly linked to DPLA).
```

Comment:
Too many asterisks in the heading here. Change to `**DPLA metadata aggregator product**.`